### PR TITLE
[API] Adder user feature

### PR DESCRIPTION
**Pull request by erolcherim**

### CODE REVIEW

File: src/main/java/ro/unibuc/hello/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @PostMapping("/api/users")
+    @ResponseBody
+    public void postUser(@RequestBody UserDTO userDTO) {
+        try {
+            userService.saveUser(userDTO);
+        } catch (Exception E) {
```
Comment (TeodoraLazaroiu) [inline src/main/java/ro/unibuc/hello/controller/UserController.java:26]: unused variable

File: src/main/java/ro/unibuc/hello/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    @GetMapping("/api/users/{id}")
+    @ResponseBody
+    public User getUser(@PathVariable String id) {
+        try {
+            return userService.getUserById(id);
+        } catch (Exception e) {
+            if (e.getMessage().equals(HttpStatus.NOT_FOUND.toString())) {
+                throw new ResponseStatusException(HttpStatus.NOT_FOUND);
+            } else {
+                throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, "Sorry bossman it's not working");
```
Comment (TeodoraLazaroiu) [inline src/main/java/ro/unibuc/hello/controller/UserController.java:40]: ¿

**Review by TeodoraLazaroiu — APPROVED** [review]

File: src/main/java/ro/unibuc/hello/controller/UserController.java (hunk 1 above)
Comment (erolcherim) [reply src/main/java/ro/unibuc/hello/controller/UserController.java:26]: If I can ignore my IDE I can ignore this comment aswell

File: src/main/java/ro/unibuc/hello/controller/UserController.java (hunk 2 above)
Comment (erolcherim) [reply src/main/java/ro/unibuc/hello/controller/UserController.java:40]: ?

**Pull request merged**